README.md

```diff
@@ -76,27 +89,36 @@ appium driver install --source npm appium-flutter-integration-driver
 - Using native driver command will directly hit the Appium UiAutomator2/XCUITest driver.
 
 For more details, refer to the documentation for each driver:
+
 - [Appium UiAutomator2 Driver](https://github.com/appium/appium-uiautomator2-driver)
 - [Appium XCUITest Driver](https://appium.github.io/appium-xcuitest-driver/latest)
 
+## Capabilities for Appium Flutter Integration Driver
+
+| Capability                | Description                                                                | Required |
+| ------------------------- |----------------------------------------------------------------------------| -------- |
+| appium:flutterServerLaunchTimeout | Time in ms to wait for flutter server to be pingable. Default is 5 seconds | No       |
```

Comment:
I would expect both timeouts to be either in seconds or missiliseconds. Having different unit is confusing